front-end/src/app/job-details/job-details.component.ts

```diff
@@ -132,15 +131,23 @@ export class JobDetailsComponent {
         }
 
         const application: Application = {
+          approved: false,
+          reviewed: false,
           email: userProfile.email ?? "",
           username: userProfile.username ?? "",
           firstName: userProfile.firstName ?? "",
           lastName: userProfile.lastName ?? "",
           positionId: this.position.positionId,
+          // @ts-expect-error The library does not support this, but it works :)
+          phoneNumber: userProfile["attributes"].phone[0],
         };
 
         return application;
       })
     );
   }
+
+  ngAfterViewInit(): void {
+    this.didApply.next();
```

Comment:
You should unsubscribe the subject when the component is destroyed

Reply:
I am using the async pipe in html, and it unsubscribes automatically